keyboards/nullbitsco/nibble/keymaps/via/keymap.c

```diff
@@ -0,0 +1,148 @@
+#include QMK_KEYBOARD_H
```

Comment:
~~nibble_encoder.h, via_extras.h needs to be moved to "bells-and-whistles" style keymaps -- there's extra stuff over and above what's needed to get a board up and running, stuff like discord mute and the like aren't going to apply to everyone~~ -- discussed on Discord directly, and I don't think QMK has a proper solution for this, but we need to investigate how to solve this going forward. Bending the requirements in the PR checklist somewhat to allow this through -- usability of the kb for users was deemed more of a benefit than requiring "bells-and-whistles" keymaps for both `default` and `via` keymaps.